Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -861,15 +861,20 @@ def _uniqueFirstLayers(layerList):
                               timeResosTag = cms.InputTag('trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModelResolution'), 
                               nSigmaDtVertex = cms.double(3) )
 phase2_timing_layer.toModify( trackValidatorStandalone,
-                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D') )
+                              label_vertex = cms.untracked.InputTag('offlinePrimaryVertices4D'),
+                              doMTDPlots = cms.untracked.bool(True) )
```

Comment:
Are these now unnecessary? (if yes, without `cms.untracked.bool` the configuration system would have complained)